### PR TITLE
OCM-9878 | ci: include volume_size for profile rosa-hcp-advanced

### PR DESCRIPTION
**Pull request by 86254860**

### CODE REVIEW

File: tests/ci/data/profiles/rosa-hcp.yaml [hunk 1]
```diff
@@ -25,6 +25,7 @@ profiles:
     oidc_config: "managed"
     auditlog_forward: true
     admin_enabled: false
+    volume_size: 150
```
Comment (yasun1) [inline tests/ci/data/profiles/rosa-hcp.yaml:28]: Let us use the edge value, which is the smallest, to do testing.

Comment (86254860) [reply tests/ci/data/profiles/rosa-hcp.yaml]: Updated to `75`.

**Comment by yasun1** [comment]
The [hcp job](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_rosa/2469/pull-ci-openshift-rosa-master-e2e-presubmits-pr-rosa-hcp-advanced/1836277588746571776) provisioning fails, please check whether it is related to your change.
